File: 2026-01/src/day_15/boj_14910/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int prv = -1000000;

    int x;
    while (cin >> x) {
        if (x >= prv) {
            prv = x;
        } else {
            cout << "Bad";
            return 0;
        }
    }

    cout << "Good";
}

File: 2026-01/src/day_15/boj_14910/Main.java
package day_15.boj_14910;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int prv = -1_000_000;
        while (st.hasMoreTokens()) {
            int x = Integer.parseInt(st.nextToken());
            if (x >= prv) {
                prv = x;
            } else {
                System.out.println("Bad");
                return;
            }
        }

        System.out.println("Good");
    }
}
